src/components/tests/text-input.components.test.js

```diff
@@ -0,0 +1,279 @@
+/* eslint-disable no-undef */
```

Comment:
You should have two files for tests. One for TextInput and another to Email Input.

Reply:
Done